package.json

```diff
@@ -31,11 +31,12 @@
     "@babel/plugin-transform-react-jsx": "7.3.0",
     "@wordpress/babel-preset-default": "4.5.0",
     "@wordpress/blob": "2.5.0",
-    "@wordpress/block-editor": "3.0.0",
+    "@wordpress/block-editor": "3.1.0",
```

Comment:
There are [no release notes for this version](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/CHANGELOG.md), so hard to tell what the consequences are. But if all tests pass, I guess we're good.

Reply:
The tests were failing in #3123, but are passing here due to the change in the helpers file.